integrations/extensions/starter-kits/magnolia/README.md

```diff
@@ -0,0 +1,124 @@
+
+The OpenAPI spec in this starter kit includes the following **example** endpoint:
+
+- `GET /delivery/tours/v1`: Search for content in the tours workspace using the v1 version of the REST endpoint
+
+**IMPORTANT NOTE:** This is an **example endpoint only** and was setup as a custom REST endpoint in the sample demo instance of Magnolia. As an end user, you should make sure to replace this with the path of your endpoint which will get invoked like `/.rest/<endpoint>/`. See the note [here](https://docs.magnolia-cms.com/product-docs/6.2/Developing/API/Delivery-API.html#_methods) for more details. Using `delivery` as a part of your end point path will allow you to have anonymous users access the `/.rest/delivery/*` path however it is recommended to have a security strategy to safeguard your content.   
+
+The Delivery endpoint for querying nodes is described in detail in the [Magnolia docs](https://docs.magnolia-cms.com/product-docs/6.2/Developing/API/Delivery-API.html#_query_nodes). Note that the `query nodes` method is the functionality that allows a user to perform full-text searches on the content stored in their instance of Magnolia.
+
+This starter kit exposes only the minimal functionality needed for simple use cases, and you will need to extend it if you want to cover more advanced ones.  For example, the API has parameters for restricting search based on the date or country that are not listed in the simple OpenAPI specification provided in the starter kit, so you would need to add these if you wanted to employ such filters.
+
+## Pre-Requisite Steps
+
+If you don't have a Delivery endpoint already setup to use with your Magnolia instance, make sure to follow the [Getting Started docs](https://docs.magnolia-cms.com/product-docs/6.2/Getting-started-with-Magnolia/Getting-started-with-REST.html) to setup one that allows you to get content from a certain workspace node.
+
+Since there is no security enabled by default on the REST endpoint, you should make sure to follow steps outlined in the [REST security](https://docs.magnolia-cms.com/product-docs/6.2/Modules/List-of-modules/REST-module/REST-security.html) section to provide restricted access to the endpoint only for an authorized user via their corresponding username/password.
+
+## Other Setup Info
+
+### Setup in a new Assistant
+
+If you want to make a _new_ Assistant using this starter kit, take the following steps:
+
+- Download the OpenAPI specification (`magnolia-search-openapi.json`) and Actions JSON file (`magnolia-search-actions.json`) in this starter kit.
+- Update the API path from `/delivery/tours/v1` to the endpoint setup by you for your Magnolia instance in the downloaded `magnolia-search-openapi.json`
+- Use the OpenAPI specification to [build a custom extension](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-build-custom-extension#building-the-custom-extension).
```

Comment:
You also need to set the `domain` server variable at this step, e.g., to `34.71.156.142:8080/magnoliaPublic`